common/culture/traditions/00_ritual_traditions.txt

```diff
@@ -733,20 +750,21 @@ tradition_mystical_ancestors = {
 		4 = philosopher.dds
 	}
 
-	is_shown = {
-		any_parent_culture_or_above = {
-			has_cultural_tradition = tradition_mystical_ancestors
-		}
-	}
+	# Warcraft
+	# is_shown = {
+	#	any_parent_culture_or_above = {
+	#		has_cultural_tradition = tradition_mystical_ancestors
+	#	}
+	# }
 
-	can_pick = { #Can pick if your culture has mystical ancestors in its history
-		custom_tooltip = {
-			text = any_ancestor_culture_was_mystical
-			any_parent_culture_or_above = {
-				has_cultural_tradition = tradition_mystical_ancestors
-			}
-		}
-	}
+	# can_pick = { #Can pick if your culture has mystical ancestors in its history
+	#	custom_tooltip = {
+	#		text = any_ancestor_culture_was_mystical
+	#		any_parent_culture_or_above = {
+	#			has_cultural_tradition = tradition_mystical_ancestors
+	#		}
+	#	}
+	# }
```

Comment:
All trigger blocks are commented so it's avaliable to everyone. It's intented?

Reply:
Yes, although it may make more sense to limit to to certain religions or cultures, especially those that venerate wild gods, practise shamanism or are obessed with Titan lore like Dwarves. I'm open to suggestions.